CIScripts/Test/NewContainer.Tests.ps1

```diff
@@ -0,0 +1,130 @@
+Param (
+    [Parameter(Mandatory=$false)] [string] $TestenvConfFile,
+    [Parameter(Mandatory=$false)] [string] $LogDir = "pesterLogs",
+    [Parameter(ValueFromRemainingArguments=$true)] $AdditionalParams
+)
+
+. $PSScriptRoot\TestConfigurationUtils.ps1
+. $PSScriptRoot\..\Testenv\Testenv.ps1
+. $PSScriptRoot\..\Testenv\Testbed.ps1
+. $PSScriptRoot\PesterLogger\PesterLogger.ps1
+
+Describe "New-Container" {
```

Comment:
Could you tag this as CI and Systest?
```
Describe "New-Container" -Tags CI, Systest {
```

Reply:
( so that Invoke-CISelfcheck executes this)

Reply:
In separate PR. We want it to pass CI without these tags.